addons/binding/org.openhab.binding.neeo/src/main/java/org/openhab/binding/neeo/handler/NeeoHandlerFactory.java

```diff
@@ -43,6 +51,8 @@
     @NonNullByDefault({})
     private NetworkAddressService networkAddressService;
 
+    private final Map<ThingUID, @Nullable ServiceRegistration<?>> discoveryServiceRegs = new HashMap<>();
```

Comment:
Did you put the `@Nullable` for the values here on purpose because you are putting values with `null` in here? or did you do that because a `get()` on a map may return `null`. I am assuming you did it because of the later and you should not do that because this would be part of external null annotations, see https://github.com/eclipse/smarthome/issues/5908

Reply:
lol - I did that because I copied it directly from the HUE thing factory and didn't give it a second of thought.  I removed it from mine  ;)